app-sso/how-to-guides/service-operators/verify-token-settings.hbs.md

```diff
@@ -0,0 +1,192 @@
+# Verify token settings
```

Comment:
**[thought, non-blocking]:** In the TOC there are sections "token settings", "token signatures" and then "verify token settings". It seems as if the last relates to the first, so why does it come last? Could it be merged into "token settings" for cohesion?

Reply:
I went back and forth on this one and didn't know which I preferred and ended up just making a decision to move forward. I'm happy to move it!